sw/device/lib/dif/dif_hmac.h

```diff
@@ -139,11 +139,17 @@ typedef enum dif_hmac_digest_result {
 typedef struct dif_hmac_config {
   /** The base address for registers in the HMAC IP. */
   mmio_region_t base_addr;
+} dif_hmac_config_t;
+
+/**
+ * Configuration for a single HMAC Transaction
+ */
+typedef struct dif_hmac_transaction_config {
```

Comment:
Isn't the correct pattern (per the naming doc) for this to be called `dif_hmac_transaction_t` and to include other transaction params, like the key?

Reply:
Explicitly this should *not* contain the key, because we're avoiding unions (which is why they have different start functions).